packages/@expo/server/src/runtime/error.ts

```diff
@@ -0,0 +1,62 @@
+export class StatusError extends Error {
+  status: number;
+  body: string;
+
+  constructor(status = 500, body?: { error?: string; [key: string]: any } | Error | string) {
```

Comment:
```suggestion
  constructor(status = 500, body?: { error?: string; [key: string]: any } | Error | string, options?: ErrorOptions) {
```

Might be nice to include the `ErrorOptions` here, so you can pass `{ cause: otherError }` if needed?